public/css/skeleton.css

```diff
@@ -610,16 +608,26 @@ there.
 */
 
 /* Larger than mobile */
-@media (min-width: 400px) {}
+@media (min-width: 400px) {
+  /* To fill */
+}
```

Comment:
I'm not sure if we'll need this section (assuming the grid classes that Skeleton has are already enough to ensure a responsive UI) so I just added those comments to not have `stylelint` show errors.